test-e2e/cip-auditor/push-audit-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# About: Triggered by the make target "image-push-cip-auditor-e2e", this script pushes
+# auditor test images to the auditor's staging location (defined in workspace_status.sh).
+# First building from Dockerfile, the container image are only designed to runs the
+# entrypoint command "cip audit" on startup.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+repo_root=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)
+
+# Inject workspace variables
+source <("${repo_root}"/workspace_status.sh inject)
+
+# Build docker container from Dockerfile.
+output=$(docker build "$repo_root" | tail -1)
+# Grab only the container ID. Expected to transform a line like
+# "Successfully built 703827382376" into "703827382376".
+container_id=$(echo "$output" | cut -d ' ' -f 3)
+# Asemble image names
+latest="${STABLE_TEST_AUDIT_STAGING_IMG_REPOSITORY}/${STABLE_IMG_NAME}-auditor-test:latest"
+stable="${STABLE_TEST_AUDIT_STAGING_IMG_REPOSITORY}/${STABLE_IMG_NAME}-auditor-test:${STABLE_IMG_TAG}"
+# Tag the pre-built image with two new names
```

Comment:
Add a period.

But also what is a pre-built image vs a built image? I think you mean to say `Assign additional tags to the image.`

Reply:
Ack.